templates/corporate/hello.html

```diff
@@ -1,562 +1,385 @@
+            <div class='appshot-1'>
+                <picture>
+                    <!-- dark theme is only with webp images, since webp support was at the same time or earlier than prefers-color-scheme https://caniuse.com/?search=prefers-color-scheme https://caniuse.com/webp -->
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-dark-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-dark-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-dark-3x.webp') }} 3x"
+                      media="(min-width: 941px) and (prefers-color-scheme: dark)"/>
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-mobile-dark-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-mobile-dark-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-mobile-dark-3x.webp') }} 3x"
+                      media="(max-width: 940px) and (prefers-color-scheme: dark)"/>
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-3x.webp') }} 3x"
+                      media="(min-width: 941px)"/>
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-mobile-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-mobile-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-mobile-3x.webp') }} 3x"
+                      media="(max-width: 940px)"/>
+                    <source class='appshot-1__img'
+                      srcset="{{ static('images/landing-page/hello/screen-1-mobile-1x.jpg') }},
+                      {{ static('images/landing-page/hello/screen-1-mobile-2x.jpg') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-mobile-3x.jpg') }} 3x"
+                      media="(max-width: 940px)"/>
+                    <img alt="" class='appshot-1__img' src="{{ static('images/landing-page/hello/screen-1-1x.jpg') }}"
+                      srcset="{{ static('images/landing-page/hello/screen-1-1x.jpg') }},
+                      {{ static('images/landing-page/hello/screen-1-2x.jpg') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-3x.jpg') }} 3x"/>
+                </picture>
+                <div class="cta-buttons">
+                    <a href="/try-zulip/">
```

Comment:
Fixed this URL to have a trailing `/`; this saves a redirect.

Reply:
Thanks